runbotics/runbotics-common/src/actions/actions.ts

```diff
@@ -11,7 +11,8 @@ export enum CloudPath {
 export enum ActionRegex {
     VARIABLE_NAME = '^[a-zA-Z0-9_]*$',
     EXCEL_WORKSHEET_NAME = '^[^\\[\\]\\*\\?\\/\\\\\\:\\|]*$',
-    EXCEL_CELL_ADDRESS = '^[a-zA-Z]+\\$?\\d+'
+    EXCEL_CELL_ADDRESS = '^[a-zA-Z]+\\$?\\d+',
```

Comment:
imo this also should be '^[A-Z]+\\$?\\d+'

Reply:
why dollar sign? and imo there is no difference between lower and upper case

Reply:
I think this pattern is completely wrong. It verifies the coordinates of a cell, e.g. C1, but also CaB123, so imo it should match a string with two substrings. The first part matches letters and the second part matches numbers. My suggestion is a pattern that looks as follows: ^[a-zA-Z]+(\d+)$

Reply:
I'd only added '+' after [A-Z].  What @marcin-depcik suggests is best solution but imo without the brackets: ^[a-zA-Z]+\d+$. Unless you want to immediately split letters and numbers which sounds like a good idea, but then imo both should be in brackets ^([a-zA-Z])+(\d+)$

did you mean that it SHOULD or SHOULDN'T match CaB123?

Reply:
> why dollar sign? and imo there is no difference between lower and upper case

the dollar sign means end of line

Reply:
I mean the address 'CaB123' should be valid input, correct me if I am wrong.